setup.py

```diff
@@ -13,7 +13,7 @@
 from setuptools import find_packages, setup  # noqa: H301
 
 NAME = "marquez-python"
-VERSION = "0.1.7"
+VERSION = "0.1.8"
```

Comment:
should the `marquez-python` version be changing at the same time? or was this unintentional?

Reply:
The flow that @wslulciuc suggested had the version change locally in the source, and be added as part of the commit that triggers the release flow by automatically applying the correct tag.

I think the idea is for this to be run from `master`, as a separate step that ups the version and publishes the artifacts w/the matching tag.

Reply:
Talked offline, this is not necessary for this PR. @ashulmanWeWork will revert.

Reply:
Yup - needed to add this for testing. Preferred to do the testing on this branch as opposed to master. Will also delete the associated tags.

Reply:
I think one approach could've been to branch off of this branch, publish that branch to github and then test it. that would hide the change from this PR

Reply:
Yep, you're right @sshah-wework. I'll keep that in mind as we make these kinds of changes.